tests/network/jumbo_frame/test_bond.py

```diff
@@ -21,13 +21,15 @@
 BOND_NAME = "jfbond1"
 BRIDGE_NAME = "brbond1"
 
-pytestmark = pytest.mark.usefixtures(
-    "skip_if_no_multinic_nodes",
-    "skip_no_bond_support",
-    "skip_when_no_jumbo_frame_support",
-    "hyperconverged_ovs_annotations_enabled_scope_session",
-    "workers_type",
-)
+pytestmark = [
+    pytest.mark.special_infra,
```

Comment:
Ditto.
Bonds tests are supposed to be available on every multi-NIC cluster, and multi-NIC is about to become a standard cluster configuration for network tests (as part of removing the programmatic skips effort).

Reply:
same, I think it was intended for jumbo frames.

Reply:
ditto